sympy/printing/ccode.py

```diff
@@ -219,8 +219,9 @@ def _print_ITE(self, expr):
         return self._print(_piecewise)
 
     def _print_MatrixElement(self, expr):
-        return "{0}[{1}]".format(expr.parent, expr.j +
-                expr.i*expr.parent.shape[1])
+        from sympy.printing.precedence import PRECEDENCE
```

Comment:
I believe this should be an import at the beginning of the file, not inside a function. Same for all other files.